app-rating/src/main/java/com/stepstone/apprating/AppRatingDialog.kt

```diff
@@ -215,6 +219,35 @@ class AppRatingDialog private constructor(private val activity: FragmentActivity
             return this
         }
 
+        /**
+         * This method sets dialog default comment text.
+         * The comment is optional.
+         *
+         * @param comment dialog's comment text
+         * @return Builder for chaining
+         * @see#setDefaultComment(int)
+         */
+        fun setDefaultComment(comment: String): Builder {
```

Comment:
Shouldn't this be `comment: String?` as the underlying property is nullable?

Reply:
No, it's not.
I check precondition below, to avoid user passing null comment.
But internal param is Nullable bacause there is other way of setting it by resId. When user pass comment by resId, the String field can stay null